docs/plotting_tools.rst

```diff
@@ -2,14 +2,59 @@
 Plotting Tools
 ##############
 
-There are `several scripts
+There are `scripts
 <https://github.com/BEAST-Fitting/beast/tree/master/beast/plotting>`_ for making
-diagnostic plots and visualizations.  Some are described here.
+diagnostic plots and visualizations.  The command needed to access them from
+anywhere with an installed version of the beast is given along with a description.
 
-- `make_ds9_region_file.py <https://github.com/BEAST-Fitting/beast/blob/master/beast/plotting/make_ds9_region_file.py>`_:
+Observation Models
+------------------
+
+- `beast plot_toothpick_details`:
+  Shows the details of the creation of the toothpick observation model including
+  the individual AST results.
+  For more details see :mod:`~beast.plotting.plot_toothpick_details`.
+
+- `beast plot_noisemodel.py`:
+  Plot the bias and uncertainty as a function of flux (similar to Figure 12 in
+  `Gordon+16 <https://ui.adsabs.harvard.edu/abs/2016ApJ...826..104G>`_).
+  Multiple noise models can be overplotted, as long as they correspond to the
+  same SED model grid.
+  For more details see :mod:`~beast.plotting.plot_noisemodel`.
+
+Fitting
+-------
+
+- `beast plot_indiv_fit`:
+  For a given star, makes a multi-panel plot that shows the PDFs and best fits
+  of each parameter, as well as an SED (similar to Figure 14 in
+  `Gordon+16 <https://ui.adsabs.harvard.edu/abs/2016ApJ...826..104G>`_).
+  For more details see :mod:`~beast.plotting.plot_indiv_fit`.
+
+Misc
+----
+
+- `plot_cmd.py`:
+  Make a color-magnitude diagram of the observations.  Inputs are the photometry
+  file (which can be a `simulation <https://beast.readthedocs.io/en/latest/simulations.html#plotting>`_)
+  and the three filters.
+  For more details see :mod:`~beast.plotting.plot_cmd`.
+
+- `beast -m beast.plotting.plot_cmd_with_fits`:
+  Similar to above, but color-coding the data points using one of the parameters
+  from the BEAST fitting.  Takes three additional inputs: a BEAST stats file,
+  the parameter to use, and whether to apply color after taking the log10 of the
+  parameter.
+  For more details see :mod:`~beast.plotting.plot_cmd_with_fits`.
+
+- `beast.plotting.make_ds9_region_file`:
   Make a ds9 region file from an input fits catalog (use `region_file_fits`) or
   a list of artificial stars (use `region_file_txt`).  Can also choose a
   column+value as a cut to set two different region colors.
+  For more details see :mod:`~beast.plotting.make_ds9_region_file`.
+
+Docs needing updating
+---------------------
```

Comment:
What about these needs updating?

Reply:
Need to put them in the format of the above ones, add them to the API for plotting (new), and categorize them.  I'll go ahead and do this now.

Reply:
awesome, thanks!